tasks/debian-jessie.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Enable backports repository
+  apt_repository:
+    repo: deb http://ftp.debian.org/debian jessie-backports main
+    state: present
+
+- name: update the apt cache
+  apt: update_cache=yes cache_valid_time=7200
+  become: yes
```

Comment:
We don't need `become:` here, do we? Otherwise, we would need it for the other tasks as well?